FX/bryanchi_FX Devices.lua

```diff
@@ -0,0 +1,24 @@
+-- @description FX Devices
+-- @author Bryan Chi
+-- @version 1.0beta
+-- @changelog
+--   --------------------------==  declare Initial Variables & Functions  ------------------------
+--       VersionNumber = 'Dec 23 2022 - bug fix 3 '
+--       FX_Add_Del
+-- @provides
+--   . > BryanChi_FX_Devices.lua
+--   bryanchi_FX Devices/FXD MainScript.lua
+--   [data] bryanchi_FX Devices/IconFont1.ttf
+--   [effect] bryanchi_FX Devices/FXD Macros.jsfx
+--   [effect] bryanchi_FX Devices/FXD Split to 32 Channels.jsfx
+--   [effect] bryanchi_FX Devices/FXD Gain Reduction Scope.jsfx
+--   bryanchi_FX Devices/FXD ReSpectrum.jsfx
+--   bryanchi_FX Devices/FXD Split To 4 Channels.jsfx
```

Comment:
These two JSFX were not set to the "JS effect" resource type (= they would be installed in the Scripts folder instead of Effects).